src/sentry/static/sentry/app/views/projectDetail/projectDetail.tsx

```diff
@@ -48,26 +55,95 @@ class ProjectDetail extends AsyncView<Props, State> {
     return routeTitleGen(t('Project %s', params.projectId), params.orgId, false);
   }
 
-  getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
-    const {params} = this.props;
+  componentDidMount() {
+    this.syncProjectWithSlug();
+  }
+
+  componentDidUpdate() {
+    this.syncProjectWithSlug();
+  }
+
+  get project() {
+    const {projects, params} = this.props;
+
+    return projects.find(p => p.slug === params.projectId);
+  }
 
-    if (this.state?.project) {
-      return [];
+  handleProjectChange = (selectedProjects: number[]) => {
+    const {projects, router, location, organization} = this.props;
+
+    const newlySelectedProject = projects.find(p => p.id === String(selectedProjects[0]));
```

Comment:
will the project details page be updated according to the first selected project?

Reply:
you can only select one on this page